frontend/src/views/BannerTrial.vue

```diff
@@ -0,0 +1,89 @@
+<template>
+  <div v-if="show" class="bg-accent">
+    <div class="mx-auto py-3 px-3">
+      <div class="flex items-center justify-between flex-wrap">
+        <div class="w-0 flex-1 flex items-center">
+          <p class="ml-3 font-medium text-white truncate">
+            <span>
+              {{
+                $t("banner.trial-expires", {
+                  plan: currentPlan,
+                  days: nextPaymentDays,
+                })
+              }}
+            </span>
+          </p>
+        </div>
+        <div
+          class="order-3 mt-2 flex-shrink-0 w-full sm:order-2 sm:mt-0 sm:w-auto"
+        >
+          <a
+            :href="extendTrialingEmail"
+            target="_self"
+            class="flex items-center justify-center pl-4 pr-2 py-2 border border-transparent rounded-md shadow-sm text-base font-medium text-accent bg-white hover:bg-indigo-50"
```

Comment:
Are different paddings for left and right intentional?

Reply:
Thanks for remind. Fixed the padding now